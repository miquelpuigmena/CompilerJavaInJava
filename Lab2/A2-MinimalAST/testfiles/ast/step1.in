int a(int x, int i) {}
int b() {}


