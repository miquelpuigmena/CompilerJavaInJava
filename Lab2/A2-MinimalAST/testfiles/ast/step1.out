Program
  List
    FuncDecl
      IdDecl ID="a"
      Arg
      Block
        Opt
    FuncDecl
      IdDecl ID="b"
      Arg
      Block
        Opt
