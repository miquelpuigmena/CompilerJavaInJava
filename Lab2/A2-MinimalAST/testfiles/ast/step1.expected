Program
  List
    FuncDecl
      IdDecl ID="a"
      Arg
        Opt
          List
            AssgmtStmt
              IdUse ID="x"
              Opt
            AssgmtStmt
              IdUse ID="i"
              Opt
      Block
        Opt
    FuncDecl
      IdDecl ID="b"
      Arg
        Opt
      Block
        Opt
